tests/kernel/mbox/mbox_api/testcase.yaml

```diff
@@ -1,3 +1,4 @@
 tests:
   kernel.mailbox:
     tags: kernel
+    platform_exclude: quark_d2000_crb nucleo_f030r8
```

Comment:
add newline

Reply:
is that because of ram? in this case please add min_ram: 16 instead of listing platforms

Reply:
fixed it.